### ISSUE TITLE
印章检测+文字识别：弯曲文字识别错误

### ISSUE THREAD

**Issue by fengzhengxiong** [PaddlePaddle/PaddleOCR#15735]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

我正在做弯曲文字识别的工作。采用的是两步法：文本检测+文本识别。文本检测我用的印章检测模型，标注完成训练后，用张新图去推理效果还可以，之后用自己的图像分割脚本，将检测到的区域裁剪成文字识别模型数据集，（即将检测文字区域按照标注点抠出来，放到一张空白的底图中），之后去训练。当我得到文字检测模型之后，就去拿图片进行推理，发现这个模型对弯曲的中文识别很差，但对弯曲的数字识别还可以。我不理解是哪个步骤出现了问题。请问哪位同学帮我指正错误，谢谢

### 🏃‍♂️ Environment (运行环境)

印章检测模型：PP-OCRv4_server_seal_det_pretrained.pdparams
文字识别模型：PP-OCRv5_server_rec_pretrained.pdparams
python : 3.10

识别弯曲文字结果： result: 威沂市瑞铁路工有限公司 0.49418675899505615
识别弯曲数字结果： result: 1744677123976  0.9686041474342346
识别不弯曲文字结果： result: 名称专用章     0.9664716720581055

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

(ppocr_env) D:\AlsonPyWork\axiongOCRTest>python tools/infer_rec.py -c configs/PP-OCRv5_server_rec.yml -o Global.infer_img=datasets/test_img/circle_Aug00014_crop_0.jpg Global.pretrained_model=output/PP-OCRv5_server_rec/save_model/latest.pdparams
信息: 用提供的模式无法找到文件。
D:\Anaconda3\envs\ppocr_env\lib\site-packages\paddle\utils\cpp_extension\extension_utils.py:711: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
[2025/06/16 10:44:25] ppocr WARNING: Skipping import of the encryption module.
[2025/06/16 10:44:25] ppocr INFO: Architecture :
[2025/06/16 10:44:25] ppocr INFO:     Backbone :
[2025/06/16 10:44:25] ppocr INFO:         name : PPHGNetV2_B4
[2025/06/16 10:44:25] ppocr INFO:         text_rec : True
[2025/06/16 10:44:25] ppocr INFO:     Head :
[2025/06/16 10:44:25] ppocr INFO:         head_list :
[2025/06/16 10:44:25] ppocr INFO:             CTCHead :
[2025/06/16 10:44:25] ppocr INFO:                 Head :
[2025/06/16 10:44:25] ppocr INFO:                     fc_decay : 1e-05
[2025/06/16 10:44:25] ppocr INFO:                 Neck :
[2025/06/16 10:44:25] ppocr INFO:                     depth : 2
[2025/06/16 10:44:25] ppocr INFO:                     dims : 120
[2025/06/16 10:44:25] ppocr INFO:                     hidden_dims : 120
[2025/06/16 10:44:25] ppocr INFO:                     kernel_size : [1, 3]
[2025/06/16 10:44:25] ppocr INFO:                     name : svtr
[2025/06/16 10:44:25] ppocr INFO:                     use_guide : True
[2025/06/16 10:44:25] ppocr INFO:             NRTRHead :
[2025/06/16 10:44:25] ppocr INFO:                 max_text_length : 25
[2025/06/16 10:44:25] ppocr INFO:                 nrtr_dim : 384
[2025/06/16 10:44:25] ppocr INFO:         name : MultiHead
[2025/06/16 10:44:25] ppocr INFO:     Transform : None
[2025/06/16 10:44:25] ppocr INFO:     algorithm : SVTR_HGNet
[2025/06/16 10:44:25] ppocr INFO:     model_type : rec
[2025/06/16 10:44:25] ppocr INFO: Eval :
[2025/06/16 10:44:25] ppocr INFO:     dataset :
[2025/06/16 10:44:25] ppocr INFO:         data_dir : datasets/ocr_curve_rec_dataset_examples
[2025/06/16 10:44:25] ppocr INFO:         label_file_list : ['datasets/ocr_curve_rec_dataset_examples/val.txt']
[2025/06/16 10:44:25] ppocr INFO:         name : SimpleDataSet
[2025/06/16 10:44:25] ppocr INFO:         transforms :
[2025/06/16 10:44:25] ppocr INFO:             DecodeImage :
[2025/06/16 10:44:25] ppocr INFO:                 channel_first : False
[2025/06/16 10:44:25] ppocr INFO:                 img_mode : BGR
[2025/06/16 10:44:25] ppocr INFO:             MultiLabelEncode :
[2025/06/16 10:44:25] ppocr INFO:                 gtc_encode : NRTRLabelEncode
[2025/06/16 10:44:25] ppocr INFO:             RecResizeImg :
[2025/06/16 10:44:25] ppocr INFO:                 image_shape : [3, 48, 320]
[2025/06/16 10:44:25] ppocr INFO:             KeepKeys :
[2025/06/16 10:44:25] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_gtc', 'length', 'valid_ratio']
[2025/06/16 10:44:25] ppocr INFO:     loader :
[2025/06/16 10:44:25] ppocr INFO:         batch_size_per_card : 128
[2025/06/16 10:44:25] ppocr INFO:         drop_last : False
[2025/06/16 10:44:25] ppocr INFO:         num_workers : 4
[2025/06/16 10:44:25] ppocr INFO:         shuffle : False
[2025/06/16 10:44:25] ppocr INFO: Global :
[2025/06/16 10:44:25] ppocr INFO:     cal_metric_during_train : True
[2025/06/16 10:44:25] ppocr INFO:     calc_epoch_interval : 1
[2025/06/16 10:44:25] ppocr INFO:     character_dict_path : datasets/ocr_curve_rec_dataset_examples/dict.txt
[2025/06/16 10:44:25] ppocr INFO:     checkpoints : None
[2025/06/16 10:44:25] ppocr INFO:     d2s_train_image_shape : [3, 48, 320]
[2025/06/16 10:44:25] ppocr INFO:     debug : False
[2025/06/16 10:44:25] ppocr INFO:     distributed : False
[2025/06/16 10:44:25] ppocr INFO:     epoch_num : 75
[2025/06/16 10:44:25] ppocr INFO:     eval_batch_step : [0, 2000]
[2025/06/16 10:44:25] ppocr INFO:     infer_img : datasets/test_img/circle_Aug00014_crop_0.jpg
[2025/06/16 10:44:25] ppocr INFO:     infer_mode : False
[2025/06/16 10:44:25] ppocr INFO:     log_smooth_window : 20
[2025/06/16 10:44:25] ppocr INFO:     max_text_length : 25
[2025/06/16 10:44:25] ppocr INFO:     model_name : PP-OCRv5_server_rec
[2025/06/16 10:44:25] ppocr INFO:     pretrained_model : output/PP-OCRv5_server_rec/save_model/latest.pdparams
[2025/06/16 10:44:25] ppocr INFO:     print_batch_step : 10
[2025/06/16 10:44:25] ppocr INFO:     save_epoch_step : 1
[2025/06/16 10:44:25] ppocr INFO:     save_inference_dir : output/PP-OCRv5_server_rec/save_inference
[2025/06/16 10:44:25] ppocr INFO:     save_model_dir : output/PP-OCRv5_server_rec/save_model
[2025/06/16 10:44:25] ppocr INFO:     save_res_path : output/PP-OCRv5_server_rec/save_res.txt
[2025/06/16 10:44:25] ppocr INFO:     use_gpu : True
[2025/06/16 10:44:25] ppocr INFO:     use_space_char : True
[2025/06/16 10:44:25] ppocr INFO:     use_visualdl : False
[2025/06/16 10:44:25] ppocr INFO: Loss :
[2025/06/16 10:44:25] ppocr INFO:     loss_config_list :
[2025/06/16 10:44:25] ppocr INFO:         CTCLoss : None
[2025/06/16 10:44:25] ppocr INFO:         NRTRLoss : None
[2025/06/16 10:44:25] ppocr INFO:     name : MultiLoss
[2025/06/16 10:44:25] ppocr INFO: Metric :
[2025/06/16 10:44:25] ppocr INFO:     main_indicator : acc
[2025/06/16 10:44:25] ppocr INFO:     name : RecMetric
[2025/06/16 10:44:25] ppocr INFO: Optimizer :
[2025/06/16 10:44:25] ppocr INFO:     beta1 : 0.9
[2025/06/16 10:44:25] ppocr INFO:     beta2 : 0.999
[2025/06/16 10:44:25] ppocr INFO:     lr :
[2025/06/16 10:44:26] ppocr INFO:         learning_rate : 0.0005
[2025/06/16 10:44:26] ppocr INFO:         name : Cosine
[2025/06/16 10:44:26] ppocr INFO:         warmup_epoch : 1
[2025/06/16 10:44:26] ppocr INFO:     name : Adam
[2025/06/16 10:44:26] ppocr INFO:     regularizer :
[2025/06/16 10:44:26] ppocr INFO:         factor : 3e-05
[2025/06/16 10:44:26] ppocr INFO:         name : L2
[2025/06/16 10:44:26] ppocr INFO: PostProcess :
[2025/06/16 10:44:26] ppocr INFO:     name : CTCLabelDecode
[2025/06/16 10:44:26] ppocr INFO: Train :
[2025/06/16 10:44:26] ppocr INFO:     dataset :
[2025/06/16 10:44:26] ppocr INFO:         data_dir : datasets/ocr_curve_rec_dataset_examples
[2025/06/16 10:44:26] ppocr INFO:         ds_width : False
[2025/06/16 10:44:26] ppocr INFO:         ext_op_transform_idx : 1
[2025/06/16 10:44:26] ppocr INFO:         label_file_list : ['datasets/ocr_curve_rec_dataset_examples/train.txt']
[2025/06/16 10:44:26] ppocr INFO:         name : MultiScaleDataSet
[2025/06/16 10:44:26] ppocr INFO:         transforms :
[2025/06/16 10:44:26] ppocr INFO:             DecodeImage :
[2025/06/16 10:44:26] ppocr INFO:                 channel_first : False
[2025/06/16 10:44:26] ppocr INFO:                 img_mode : BGR
[2025/06/16 10:44:26] ppocr INFO:             RecAug : None
[2025/06/16 10:44:26] ppocr INFO:             MultiLabelEncode :
[2025/06/16 10:44:26] ppocr INFO:                 gtc_encode : NRTRLabelEncode
[2025/06/16 10:44:26] ppocr INFO:             KeepKeys :
[2025/06/16 10:44:26] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_gtc', 'length', 'valid_ratio']
[2025/06/16 10:44:26] ppocr INFO:     loader :
[2025/06/16 10:44:26] ppocr INFO:         batch_size_per_card : 128
[2025/06/16 10:44:26] ppocr INFO:         drop_last : True
[2025/06/16 10:44:26] ppocr INFO:         num_workers : 16
[2025/06/16 10:44:26] ppocr INFO:         shuffle : True
[2025/06/16 10:44:26] ppocr INFO:     sampler :
[2025/06/16 10:44:26] ppocr INFO:         divided_factor : [8, 16]
[2025/06/16 10:44:26] ppocr INFO:         first_bs : 128
[2025/06/16 10:44:26] ppocr INFO:         fix_bs : False
[2025/06/16 10:44:26] ppocr INFO:         is_training : True
[2025/06/16 10:44:26] ppocr INFO:         name : MultiScaleSampler
[2025/06/16 10:44:26] ppocr INFO:         scales : [[320, 32], [320, 48], [320, 64]]
[2025/06/16 10:44:26] ppocr INFO: profiler_options : None
[2025/06/16 10:44:26] ppocr INFO: train with paddle 3.0.0 and device Place(gpu:0)
W0616 10:44:26.024087  2248 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 12.6, Runtime API Version: 11.8
W0616 10:44:26.024087  2248 gpu_resources.cc:164] device: 0, cuDNN Version: 8.9.
[2025/06/16 10:44:26] ppocr INFO: load pretrain successful from output/PP-OCRv5_server_rec/save_model/latest
[2025/06/16 10:44:26] ppocr INFO: infer_img: datasets/test_img/circle_Aug00014_crop_0.jpg
[2025/06/16 10:44:27] ppocr INFO:        result: 威沂市瑞铁路工有限公司 0.49418675899505615
[2025/06/16 10:44:27] ppocr INFO: success!
